assets/dev/js/editor/controls/media.js

```diff
@@ -43,6 +46,21 @@ ControlMediaItemView = ControlMultipleBaseItemView.extend( {
 		return ( 'svg' === mediaType ) ? 'image/svg+xml' : mediaType;
 	},
 
+	updatePrevState( options ) {
+		if ( ! options.id ) {
+			return;
+		}
+		this.setEditSetting( 'mediaId', options.id );
+		this.setEditSetting( 'mediaAlt', options.alt );
+		this.setEditSetting( 'mediaSize', options.size );
+	},
+
+	resetPrevState() {
+		this.setEditSetting( 'mediaId', null );
+		this.setEditSetting( 'mediaAlt', null );
+		this.setEditSetting( 'mediaSize', null );
+	},
```

Comment:
https://github.com/elementor/elementor/blob/cf465842dd6e009cd50847f4ed2b8e289b875a10/assets/dev/js/editor/controls/base-data.js#L175

Which problem it will resolve?